package.json

```
@@ -111,6 +111,7 @@
     "balanced-match": "^2.0.0",
     "colord": "^2.9.2",
     "cosmiconfig": "^7.0.1",
+    "css-functions-list": "^2.0.0",
```

[suggest] Bump up, please?

```suggestion
    "css-functions-list": "^3.0.0",
```

https://github.com/niksy/css-functions-list/releases/tag/v3.0.0